src/Microsoft.CodeQuality.Analyzers/Core/QualityGuidelines/MicrosoftQualityGuidelinesAnalyzersResources.resx

```diff
@@ -246,4 +246,13 @@
   <data name="MarkMembersAsStaticCodeFix_WarningAnnotation" xml:space="preserve">
     <value>Some references to '{0}' could not be fixed, they should be fixed manually.</value>
   </data>
+  <data name="AvoidPropertySelfAssignmentTitle" xml:space="preserve">
+    <value>Do not assign properties to themselves.</value>
```

Comment:
Doesn't look grammatically correct. @gewarren for suggestion on the title and also the message and description below.

Reply:
Yes, this is really just some placeholder text until I could wordsmith it into something more correct. Any help/suggestions from @gewarren would be appreciated!

Reply:
It doesn't sound too bad to me but perhaps:

"Do not assign a property to itself."